CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Changelog
+
+## [Unreleased]
+
+### Changed
+
+### Added
+
+### Fixed
+
+## [0.0.1] - 2025-08-14
```

Comment:
It's not a bugfix, so I would say it is v0.1.0 (with the v in front - maybe that's just me)